lib/cmd/common/admin-devices.js

```diff
@@ -24,28 +24,28 @@ function devices (argv, cb) {
   var action = args[0];
   if (action === 'list') {
     return list(cb);
-  }else if (action === 'read'){
```

Comment:
And it's better to refactor this else if's blocks to something better designed ))